designs/virtualization/4.5/Scheduling/Tolerations/tolerations.md

```diff
@@ -0,0 +1,25 @@
+---
+parent: virtualization
+version: 4.5
+---
+
+# VM Tolerations
+
+Managing VM tolerations for tainted nodes
+
+![image name](img/Tolerations-0-0.jpg)
+Within the VM page, Details tab, the Tolerations field is placed under the Scheduling section.
+
+![image name](img/Tolerations-1-0.jpg)
```

Comment:
Related to @andybraren comment about the notification here. We should align with PF and not create a new style for the inline alerts.

Reply:
Resolved in the [Node-selector PR](https://github.com/openshift/openshift-origin-design/pull/371)